### PR TITLE
Staging fixes22.2

### PR DESCRIPTION
**Pull request by Bod9001**
adds a new component that animates light sprites

### Changelog:

CL: [Fix] Fixes eggs sometimes not hatching
CL: [Balance] syringe has text when starting injection
CL: [Balance] baby dinosaurs are bigger
CL: [Fix]  Body parts now get Thrown out If they can't Be accessed in the Parent that's being morphed
CL: [Fix] fixes cyborgs having body parts
CL: [Fix] fixes cyborgs having mutations

### CODE REVIEW

File: UnityProject/Assets/Scripts/Core/Sprite Handler/LightSpriteHandler.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
+using Light2D;
+using UnityEngine;
+
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Core/Sprite Handler/LightSpriteHandler.cs:3]: adding a new class without namespace

File: UnityProject/Assets/Scripts/Messages/Client/RequestCableCut.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+using Messages.Client;
+using Mirror;
+using UnityEngine;
+
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Messages/Client/RequestCableCut.cs:3]: another new class without namespace

**Review by corp-0 — CHANGES_REQUESTED** [review]
other than wondering why the interaction message on tiles was removed, I don't have much to say.

Add the namespaces and we are done.

**Comment by Bod9001** [comment]
Because it had a register handler And I'm wondering is it that's causing the weird mirror issue, 
And also it's not like any other Interaction messages so is better to make it standard

**Pull request merged**